doc/_sphinx/theme/doctools.js

```diff
@@ -21,11 +21,18 @@ const _ready = (callback) => {
   }
 };
 
+const BLACKLISTED_KEY_CONTROL_ELEMENTS = new Set([
+  "TEXTAREA",
+  "INPUT",
+  "SELECT",
+  "BUTTON",
+]);
+
 /**
  * highlight a given string on a node by wrapping it in
  * span elements with the given class name.
  */
-const _highlight = (node, addItems, text, className, index) => {
+const _highlight_flames = (node, addItems, text, className, index) => {
```

Comment:
How come it needed a rename? And shouldn't this be singular if the rename is needed?

Reply:
In short, the docs are in this hybrid theme land where they are using some features from one version but need a later version for other features that were then upgraded in the latest version.  The hybrid naming convention keeps the functions separate so they don't conflict and keep the functionality for both old and new.

If there is a desire to completely overhaul the theme, I can go do that as well and then we some of these files would not be needed anymore.